### PR TITLE
Work around an OSX xcode ld error temporarily

### PR DESCRIPTION
**Pull request by smarterclayton**
Can remove once upstream fixes

[merge] @stevekuznetsov

### CODE REVIEW

File: hack/common.sh [hunk 1]
```diff
@@ -231,11 +231,17 @@ os::build::internal::build_binaries() {
...
+      if [[ "${platform}" == "darwin/amd64" ]]; then
+        local_ldflags+=" -s"
+      fi
+
       if [[ ${#nonstatics[@]} -gt 0 ]]; then
         GOOS=${platform%/*} GOARCH=${platform##*/} go install \
           -pkgdir "${OS_OUTPUT_PKGDIR}/${platform}" \
           -tags "${OS_GOFLAGS_TAGS-} ${!platform_gotags_envvar:-}" \
-          -ldflags "${version_ldflags}" \
+          -ldflags="${local_ldflags}" \
```
Comment (stevekuznetsov) [inline hack/common.sh]: spurious `=` but ok

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**